packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -72,6 +72,12 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
   });
 
   describe('Alert Long Text', () => {
+    it('focuses on show more button', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/alert/long-text-alert');
+      browser.keys(['Tab']);
+      Terra.validates.element('show more button focused');
```

Comment:
we should add a programmatic test for focus.

Reply:
There is update to the tests: https://github.com/cerner/terra-core/pull/3889/commits/fcfbb3334033048c261b10bc89941bc6578a132d
Is that it, or need some other improvements? If more tests needed, the examples would be very appreciated.

Reply:
That looks good!